src/bidiMapper/domains/context/bidiPreloadScript.ts

```diff
@@ -116,6 +116,7 @@ export class BidiPreloadScript {
       'Page.addScriptToEvaluateOnNewDocument',
       {
         source: this.#getEvaluateString(),
+        runImmediately: true,
```

Comment:
should not some `Runtime.evaluate` calls be removed with this? Also, not sure if the pinned browser already supports that?

Reply:
> Also, not sure if the pinned browser already supports that?

Ah, of course. We will probably need to wait for the roll up.

Reply:
> should not some Runtime.evaluate calls be removed with this?

Yes. Done. Was writing a test first